WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -28101,7 +28038,6 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				INFOPLIST_FILE = "WordPressIntents/Supporting Files/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
This means that this PR will regress the deployment target of this target from 14 to 13 (and similar for many of the Widgets targets that are in the same situation).

This is what led to the CI failing on this PR, because some APIs in those targets are only available in iOS14+

Reply:
Great catch, thanks!

I initially pushed this branch with the iOS 14 bump from #19469. When I saw CI fail, I assumed it was due to the errors resulting from the bump and moved the commit in the branch that became #19469, but didn't check back here.

> I say let's do everything in one single PR

Yeah. It would have been good to get this out of the way in isolation just for the sake of the diff. The only benefit it brings is making updating that value simpler, which is what that other PR does, no need to land it into `trunk` before that.